### PR TITLE
Add: Ticket On Fly and QR attach ✅

### LINKED ISSUE #5: Add Tickets on fly :)
- [x] Create tickets on fly
- [ ] get data on fly and make qr from them
- [x] attach the qr to tickets
- [ ] Send the tickets to RSVP's person

### PR DESCRIPTION
**Pull request by nimit2801**
Closes #5

### CODE REVIEW

**Comment by nimit2801** [comment]
please review @shreyaparadkar thanks <3

File: server.js [hunk 1]
```diff
@@ -20,20 +22,30 @@ const sender = async (to, user_name) => {
...
+        path: './onfly.png',
+      },
+    ],
 
     onError: (e) => console.log(e),
     onSuccess: (i) => console.log(i),
   });
 };
-
+// sender('uid123', 'ashwin', 'ashwinadiga0111@gmail.com');
```
Comment (shreyaparadkar) [inline server.js]: Remove personal details

Comment (nimit2801) [reply server.js]: Sure would do!

File: src/tickets-onfly/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { createCanvas, loadImage } from 'canvas';
+import fs from 'fs';
+import { makeQROnFly } from './makeQR.js';
+const canvas = createCanvas(4096, 1379);
+const ctx = canvas.getContext('2d');
+
+export const createTicketOnFly = async (uid, name, email) => {
+  // Write "Awesome!"
```
Comment (shreyaparadkar) [inline src/tickets-onfly/index.js]: Update comments

File: src/tickets-onfly/makeQR.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+export const makeQROnFly = async (uid, name, email) => {
+  const token = jwt.sign({ uid, name, email }, '%hacktoberfest2022%%');
+  QRCode.toCanvas(canvas, token, function cb(err) {
+    const buffer = canvas.toBuffer('image/png');
+    fs.writeFileSync('./qr.png', buffer);
+  });
+};
+
+// example
+// makeQROnFly('asdasda123', 'nimit', 'mnimitsavant@gmail.com');
```
Comment (shreyaparadkar) [inline src/tickets-onfly/makeQR.js]: Remove personal details

**Review by shreyaparadkar — APPROVED** [review]
LGTM 🚀

File: src/tickets-onfly/makeQR.js (hunk 3 above)
Comment (nimit2801) [reply src/tickets-onfly/makeQR.js]: done!

**Pull request merged**